src/screens/auth/ForgotPasswordScreen.tsx

```diff
@@ -48,14 +45,8 @@ export default function ForgotPasswordScreen({
 
   return (
     <LoginContainer>
-      <StandardHeader alignment="flex-start">
-        <BackButtonContainer>
-          <RootNavBackButton onPress={() => navigation.navigate('Login')}>
-            <DarkGrayText>
-              <Icon name="left" size={14} color={Colors.darkGray} /> Back
-            </DarkGrayText>
-          </RootNavBackButton>
-        </BackButtonContainer>
+      <StandardHeader>
+        {BackButton(() => navigation.navigate('Login'))}
```

Comment:
This is not how you use a React component—it should always be used with the JSX syntax (`<BackButton onPress={...} />`. Here, you're just calling the component as a function. Resources on why not to do this:

https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l
https://kentcdodds.com/blog/dont-call-a-react-function-component